data/map planets.txt

```diff
@@ -4773,6 +4806,10 @@ planet "Varu Ek'lak'lai"
 	outfitter "Wanderer Basics"
 	"required reputation" 15
 	bribe 0
+	tribute 1300
+		threshold 3400
+		fleet "Wanderer Flycatchers" 20
```

Comment:
Why does only this planet have flycatchers in the fleet? Also, couldn't this spawn them before they exist?

Reply:
Doesn’t it have a Wanderer Defense fleet as well? I can take these out though if they don’t exist from the beginning of the game.

Reply:
The Flycatcher is added in the event `"wanderers: first tech increase"`, which is also the event that adds the Wanderer AM that the Flycatcher uses. Until that event, the Flycatcher doesn't exist.

Reply:
I’ll remove them entirely then. Should I also add another Wanderer Defense fleet?